### ISSUE TITLE
The example idioms seem like they need to be updated to CSD01

### ISSUE THREAD

**Issue by johnwunder** [oasis-open/cti-stix2-json-schemas#24]
It looks like several of the idioms need to be updated from RC4. In particular the Yara and Snort examples should be removed because that's no longer valid.

Also, did/can we set up CI to auto-validate these?

**Comment by gtback** [comment]
Agreed that some need to be updated/removed.

We don't currently validate the examples as part of CI. It's a bit of a chicken and egg problem (validating the examples in this repo requires the validator, but the validator depends on the schemas in this repo). It's not impossible to do, but requires a bit of a dance to update the repositories in parallel when the specs change. I'll create a new issue to set up CI.

**Comment by gtback** [comment]
@ATweedMITRE: let me know if/when the examples are updated for CSD01. They all pass the validator currently, but in case there's anything else that needs to be changed, I'll leave this open.

**Comment by ATweedMITRE** [comment]
I've removed some of the unnecessary ones that didn't quite fit for 2.0 and will save some for 2.1. The rest of what I could find was just issues with STIX patterning but everything else seems up-to-date with CSD01.
